### DISCUSSION TITLE
How to switch back to addons.mozilla.org version of sidebery from github version ?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [mbnuqw/sidebery#1248] [Q&A] [answered]
What is the difference between -> getting sidebery from addons.mozilla.org or from github ?

Currently i am on github (v5.0.0.1) version , how can i switch  to addons.mozilla version without loosing anything ?

**Comment by mbnuqw** [comment, marked as answer]
> how can i switch to addons.mozilla version without loosing anything?

- Create snapshot
- Create full backup (Sidebery settings >> Help >> Export)
- Remove addon on `about:addons` and reload the page (this will reset addon storage)
- Go to `https://addons.mozilla.org/en-US/firefox/addon/sidebery/` and install addon
- Import backup
- Restore snapshot

<br>

> What is the difference between -> getting sidebery from addons.mozilla.org or from github ?

The version in addons.mozilla.org manually reviewed by Mozilla staff, which takes ~2-3 days. The version in github is the same build but signed for self distribution.

**Reply by KAGEYAM4** [reply]
I used this https://addons.mozilla.org/firefox/downloads/file/4170134/sidebery-5.0.0.xpi provided on top of readme, and just installed it and didn't had to remove the addon from `about:addons`. 

Is this correct or should i follow your above steps ?
